contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -603,19 +634,19 @@
         }
 
         const formulaEl = document.createElement('span');
+        formulaEl.setAttribute('is', 'markdown-formula');
+        formulaEl.setAttribute('editing', true);
         formulaEl.innerHTML = formula;
-        formulaEl.classList.add(CLASS_MATH_FIELD, CLASS_MATH_FIELD_NEW);
-
+        let formulaHTML = formulaEl.outerHTML;
         const activeMathFieldEl = this.findActiveMathField();
 
         if (activeMathFieldEl !== null) {
-          activeMathFieldEl.parentNode.replaceChild(formulaEl, activeMathFieldEl);
+          activeMathFieldEl.outerHTML = formulaHTML;
```

Comment:
I found that setting `outerHTML` was the simplest way to get the custom element to initialize.